pkg/search/controller.go

```diff
@@ -410,7 +410,7 @@ func (c *Controller) deleteCluster(obj interface{}) {
 	c.queue.Add(cluster.GetName())
 }
 
-// getClusterAndResource returns the cluster and resources from the resourceRegistry object
+// getClusters return the cluster from the resourceRegistry object
```

Comment:
return -> returns

Reply:
@XiShanYongYe-Chang The code has been modified.